lib/runtime-c-api/src/import/mod.rs

```diff
@@ -661,6 +668,24 @@ pub unsafe extern "C" fn wasmer_import_func_new(
     Box::into_raw(export) as *mut wasmer_import_func_t
 }
 
+/// Hello
+#[no_mangle]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe extern "C" fn wasmer_import_trap(
+    ctx: *const wasmer_instance_context_t,
+    error_message: *const c_char,
+) -> c_uint {
+    let ctx = &*(ctx as *const Ctx);
+    let error_message = CStr::from_ptr(error_message).to_str().unwrap();
```

Comment:
`CStr::from_ptr` calls `slice::from_raw_parts` without checking if the pointer is null, [from raw parts](https://doc.rust-lang.org/std/slice/fn.from_raw_parts.html) says that the pointer must not be null; we should probably check it on our side then

Reply:
Thanks. It's a WIP ;-).